### PR TITLE
LPAL-255 Fix email text overflow in help/guidance popup

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

Fix accessibility issue where email address on the guidance popup (at the bottom of the popup) overflowed the text box.

Fixes LPAL-255

## Approach

Allow words to break within the guidance popup. This causes the email address to wrap within the text box.

## Learning

It's not possible to test browser zoom in JS. Instead, I made the viewport really narrow to force the text to wrap. I then applied the technique of comparing scrollWidth to clientWidth, to ensure that the clientWidth was at least as big as the scrollWidth (i.e. the text in the block element doesn't need scrollbars, therefore it is either enclosed in the textbox or wrapping).

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [x] The product team have tested these changes

### CODE REVIEW

File: service-front/assets/sass/patterns/_help-system.scss [hunk 1]
```diff
@@ -185,6 +185,12 @@
 .help-footer {
     padding: $gutter 0 0;
 
+    p {
+        overflow: hidden;
```
Comment (williamfalconeruk) [inline service-front/assets/sass/patterns/_help-system.scss:189]: nice bit of CSS - how i miss messing with this stuff 😁

**Review by williamfalconeruk — COMMENTED** [review]
Looks ok to me!

**Review by williamfalconeruk — APPROVED** [review]
Approved

File: service-front/assets/sass/patterns/_help-system.scss (hunk 1 above)
Comment (townxelliot) [reply service-front/assets/sass/patterns/_help-system.scss:189]: I could probably just add break word and not the other styles, but these don't hurt as they only affect this one paragraph and help it display better where the word breaks aren't supported (e.g. very old browsers).

Comment (williamfalconeruk) [reply service-front/assets/sass/patterns/_help-system.scss:189]: makes sense

**Comment by townxelliot** [comment]
I'll release this tomorrow morning and update from master then, too.

**Pull request merged**